exercises/practice/darts/example.cpp

```diff
@@ -0,0 +1,20 @@
+#include "example.h"
+#include <cmath>
+
+namespace darts {
+
+    int score(float x, float y) {
+        auto dart_location = sqrt(x*x + y*y);
```

Comment:
- Officially you need the prefix `std::` when you call a function from `<cmath>` like `sqrt()`. The major implementations of the standard library also define `::sqrt()` in `<cmath>` but that's not guaranteed by the standard.
- `dart_location` is not really a "location" but a "distance" between the center and the dart. I'd suggest a variable name like `distance`.
- Instead of `std::sqrt(a * a + b * b)` you can call `std::hypot(a, b)`. IMHO that would express the intent a little bit better.